portfolio/src/main/java/com/google/sps/servlets/listImages.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/list-images")
+public class listImages extends HttpServlet{
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+        //list all of the uploaded files
+
+        String projectId = "mluna-sps-summer22";
```

Comment:
For constant values that won't ever change, like these, it's generally good practice to create it as a constant class variable. The way to do that is to declare it with the key words `static final`, so these would go up in like line 16 as `private static final String PROJECT_ID = ... ` (it's also usually nice to have constant variables be in all uppercase, for easier reading). 

Here's a good link as to why we do this: https://www.theserverside.com/video/Why-we-use-static-final-in-Java-for-constants